.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: php
+
+php:
+  - 7.3
```

Comment:
Just a question - have you tried any newer versions?
Given 7.3 is no longer in support, 7.4 is just ending now too! 😭

Reply:
just tried this https://app.travis-ci.com/github/may-den/sensu-dashboard/jobs/587720326
8.0 and 8.1 fail but 7.4 works :) [1ce6671](https://github.com/may-den/sensu-dashboard/pull/30/commits/1ce667124baa4ff5f8e4fdbc9420b94aa2ed922e)